LibSource/Oscillator.h

```diff
@@ -68,7 +68,7 @@ class Oscillator : public SignalGenerator {
 template<class T, class BaseOscillator=Oscillator, typename Sample=float>
 class OscillatorTemplate : public BaseOscillator {
 protected:
-  float mul;
+  float mul = 1;  // Prevent having NaN as default getFrequency result
```

Comment:
I'm not sure if that's the best solution, but previous code was breaking oscillator test because it expected getFrequency to return 0 after constructor is called. The failure was happening here:

https://github.com/pingdynasty/OwlProgram/blob/develop/TestPatches/SineOscillatorTestPatch.hpp#L16

I suppose it may depend on compiler version used.